why-apollo-advantages-disadvantages-alternatives.md

```diff
@@ -51,93 +51,90 @@ There are plenty of popular and tech savvy companies out there who are using Apo
 
 {{% sub_chapter_header "Apollo's Documentation" "apollo-documentation" %}}
 
-Even though Apollo is evolving all the time, the team and community behind it makes sure to keep the documentation up to date. Going through their documentation should give you plenty of insights into how you can build your first application with Apollo without reading up any third-party resources. They cover so many areas that it can be also overwhelming for a newcomer. So make sure you go through the content topic by topic and apply it in your own application.
+While Apollo continues to evolve, the team and community behind it keeps the documentation up to date, and the there are plenty of insights on how to build that first application. In fact, they cover so many areas it can be overwhelming for newcomers. 
 
 {{% sub_chapter_header "Apollo Libraries" "apollo-libraries" %}}
 
-Apollo offers plenty of libraries for implementing your almost perfect GraphQL tech stack for your JavaScript applications. They open source the libraries in a way so that they can be used in a composable and exchangeable way.
+Apollo offers plenty of libraries for implementing an effective GraphQL tech stack for JavaScript applications, and their libraries are open-sourced to be more manageable.  {{% a_blank "Apollo Link" "https://www.apollographql.com/docs/link/" %}} provides an API for chaining different features into a GraphQL control flow. This makes it possible for automatic network retries or RESTful API endpoints instead of a GraphQL endpoints (the endpoints can be used together. too).
```

Comment:
"{{% a_blank "Apollo Link" "https://www.apollographql.com/docs/link/" %}}" -> "For instance, {{% a_blank "Apollo Link" "https://www.apollographql.com/docs/link/" %}}"

Reply:
I added this, but keep in mind, the phrase "for instance" is almost always overused.

Reply:
Ok 👍  Just wanted to point out here that Apollo Link is only **one** of the libraries.